docs/source/getting-started/content/con_understanding-codeready-containers.adoc

```diff
@@ -1,7 +1,7 @@
 [id="understanding-codeready-containers_{context}"]
 = Understanding {prod}
 
-{rh-prod} brings a minimal OpenShift 4.0 or newer cluster to your local computer.
+{rh-prod} brings a minimal OpenShift 4.1 or newer cluster to your local computer.
```

Comment:
Or just "brings a minimal OpenShift 4 cluster to your local computer." ?

Reply:
I agree with Christophe here (and I believe I made the same suggestion in the past at some point).

Reply:
@robin-owen @cfergeau Done 👍